Pipfile

```diff
@@ -15,6 +15,10 @@ gensim = "*"
 kneed = "*"
 sklearn = "*"
 papermill = "*"
+black = "*"
```

Comment:
```suggestion

```
Remove this maybe, just having `allow_prereleases` should do the trick.

Reply:
It's not used directly by solver-errors-reporter, but it is `papermill's` dependency. 
Thanks, I will try doing that.